### PR TITLE
allowing number of replicas to be configurable

### PR DESCRIPTION
**Pull request by geethub97**
Resolves: https://github.com/rancher/helm-project-operator/issues/42

adding the ability to configure replicas in Helm Project Operator

Related PR: https://github.com/rancher/helm-locker/pull/12

### CODE REVIEW

File: charts/helm-project-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -6,7 +6,9 @@ metadata:
   labels: {{ include "helm-project-operator.labels" . | nindent 4 }}
     app: {{ template "helm-project-operator.name" . }}
 spec:
-  replicas: 1
+  {{- if .Values.replicas }}
+   replicas: {{ .Values.replicas }}
```
Comment (aiyengar2) [inline charts/helm-project-operator/templates/deployment.yaml:10]:
```suggestion
  replicas: {{ .Values.replicas }}
```

Comment (aiyengar2) [reply charts/helm-project-operator/templates/deployment.yaml:10]: extra space

Comment (geethub97) [reply charts/helm-project-operator/templates/deployment.yaml]: sadness -- it's always the spaces that get me

File: charts/helm-project-operator/values.yaml [hunk 2]
```diff
@@ -126,6 +126,8 @@ nameOverride: ""
 
 namespaceOverride: ""
 
+replicas: 1
```
Comment (aiyengar2) [inline charts/helm-project-operator/values.yaml:129]: bump the chart version

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**